frontend/public/components/utils/kebab.tsx

```diff
@@ -452,7 +452,7 @@ export const getExtensionsKebabActionsForKind = (kind: K8sKind) => {
 };
 
 export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
-  const { actions, kindObj, resource, isDisabled, customData } = props;
+  const { actions, kindObj, resource, isDisabled, customData, hoverMessage } = props;
```

Comment:
Since we are doing a console wide change we should look into having a default message as well. Something like `Resource is undergoing termination`. cc @spadgett

Reply:
ACK will add a default value.

Reply:
Added a default value based on the message we get from oc in such instances: `Resource is being deleted.`